python/gui/MaaS/cbv/DMODProxy.py

```diff
@@ -16,6 +16,8 @@
 from pathlib import Path
 from typing import List, Optional, Tuple, Type
 
+GUI_STATIC_SSL_DIR = Path('/usr/maas_portal/ssl')
```

Comment:
we might want to mark this as a TODO in the future to make this configurable.

Reply:
I think this should also be handled by Django as well. Shouldn't need to be too necessary. Should be in `settings.py`

Reply:
For some reason I thought I put that in settings.py.